packages/repository/test/unit/repositories/legacy-juggler-bridge.unit.ts

```diff
@@ -177,6 +177,16 @@ describe('DefaultCrudRepository', () => {
     expect(notes[0].content).to.eql('c5');
   });
 
+  it('implements Repository.updateAll() without a where object', async () => {
+    const repo = new DefaultCrudRepository(Note, ds);
+    await repo.create({title: 't3', content: 'c3'});
+    await repo.create({title: 't4', content: 'c4'});
+    const result = await repo.updateAll({content: 'c5'});
+    expect(result).to.eql(2);
+    const notes = await repo.find({where: {title: 't3'}});
```

Comment:
IIUC, when no `where` object is provided, the update is applied to all records. Please rework the assertion to verify that fact - fetch all records and verify that each one has the same content `c5`.

```ts
const notes = await repo.find();
const titles = notes.map(n => `${n.title}:${n.content}`);
expect(titles).to.deepEqual(['t3:c5', 't4:c5']);
```

The reasoning behind my example: When the assertion fails, we want to see which records identified by their title had a wrong content value. We could use `n.toObject()`, but that would add too much noise to the compared data.